llvm/include/llvm/Transforms/Scalar.h

```diff
@@ -153,9 +153,16 @@ extern char &InferAddressSpacesID;
 
 //===----------------------------------------------------------------------===//
 //
-// TLSVariableHoist - This pass reduce duplicated TLS address call.
+// LowerExpectIntrinsics - Removes llvm.expect intrinsics and creates
+// "block_weights" metadata.
+FunctionPass *createLowerExpectIntrinsicPass();
+
+//===----------------------------------------------------------------------===//
+//
+// LowerConstantIntrinsicss - Expand any remaining llvm.objectsize and
+// llvm.is.constant intrinsic calls, even for the unknown cases.
 //
-FunctionPass *createTLSVariableHoistPass();
+FunctionPass *createLowerConstantIntrinsicsPass();
```

Comment:
Some unexpected changes here? I'd have expected this to only drop the declaration, but you are replacing it with other ones?

Reply:
Sorry, that was a mistake made during merging conflicts.